packages/bsky/src/api/app/bsky/actor/getProfile.ts

```diff
@@ -31,6 +34,10 @@ export default function (server: Server, ctx: AppContext) {
         throw new InvalidRequestError('Profile not found')
       }
 
+      if (actorClock !== null) {
+        res.setHeader('atproto-clock', actorClock)
```

Comment:
In HTTP/2, header will get lower-cased anyways, but shouldn't we `Atproto-Clock` for anybody looking at HTTP/1?

:heavy_plus_sign: to not having an `X-` prefix, as no longer encouraged as a best practice even for experimental stuff.

I would maybe bikeshed to `Atproto-Repo-Clock` or `Atproto-Self-Clock` to clarify semantics, but not a strong opinion, short+simple is nice as well.

Reply:
good call - upper cased it 👍 

hmm i could see us possibly sending multiple clocks down so not opposed to being a bit more specific in the name 🤔 
lemme think on it